include/cantera/oneD/StFlow.h

```diff
@@ -20,12 +20,13 @@ namespace Cantera
 
 // Offsets of solution components in the solution array.
 const size_t c_offset_U = 0; // axial velocity
-const size_t c_offset_V = 1; // strain rate
+const size_t c_offset_V = 1; // Alex: changing name to dimensionless radial velocity
```

Comment:
Please do not add your name to comments in the code.

Reply:
Sorry, we left these name when didn't realize. Would you be making any changes or should I need to address all the comments?

Reply:
@axbriones You should address the comments. We can give advice about the best way to update all of this.

Reply:
@axbriones The method to address the changes is:
1.) Clone your repo to your local computer: git clone https://github.com/axbriones/cantera.git
2.) Make the changes that are requested on your master branch(this will be the default branch when you clone). 
    Try to make 1 change at a time. 
    Once you've finished an adjustment for a single change request, do a:
                      git add -u   and 
                     git commit -m "I have made these changes"
3.) Follow step number 2 for all changes
4.) Do a "git push origin master"
5.) These changes will immediately be reflected in this pull request because github auto-updates pull requests with any changes made to the branch under consideration.
6.) Repeat this process as additional change requests are made.

@bryanwweber Would you add anything to this? @axbriones Is a new Github user.